extensions/amp-story/1.0/amp-story.js

```diff
@@ -1090,28 +1131,37 @@ export class AmpStory extends AMP.BaseElement {
   }
 
   /**
-   * Displays the unsupported browser UI: either the publisher provided UI, or
-   * fallbacks to a generic default.
+   * Displays changes to the support state of the browser
```

Comment:
Let's be more specific here.

> If browser is supported, displays the story. Otherwise, shows either the default unsupported browser layer or the publisher fallback (if provided).